{{cookiecutter.slug}}/frontend.angular/src/app/menu/menu.component.html

```diff
@@ -5,16 +5,17 @@
                 {{cookiecutter.project_title}}
             </strong>
 
-            <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false"
-                data-target="navbarBasicExample" (click)="toggleBurger()"
-                [ngClass]="{'is-active': burgerActive}">
+            <button class="navbar-burger burger" aria-label="menu"
+            [attr.aria-expanded]="burgerActive" data-target="navigation-links"
```

Comment:
`@aria-expanded` should be `'true'` or `'false'`. So e.g.:
 `burgerActive == 'show' ? 'true' : 'false'`.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded#values